pkgs/development/python-modules/ormar/default.nix

```
@@ -0,0 +1,82 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+, databases
+, pydantic
+, sqlalchemy
+, asyncpg
+, psycopg2
+, aiomysql
+, aiosqlite
+, cryptography
+, orjson
+, typing-extensions
+, importlib-metadata
+, aiopg
+, mysqlclient
+, pymysql
+, pytestCheckHook
+, pytest-cov
+, codecov
+, pytest-asyncio
+, fastapi
+}:
+
+buildPythonPackage rec {
+  pname = "ormar";
+  version = "0.10.23";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "collerek";
+    repo = "ormar";
+    rev = version;
+    sha256 = "sha256-ILJvJyd56lqlKq7+mUz26LvusYb5AOOfoA7OgNq2fT0=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    databases
+    pydantic
+    sqlalchemy
+    asyncpg
+    psycopg2
+    aiomysql
+    aiosqlite
+    cryptography
+
+    orjson
+    typing-extensions
+    importlib-metadata
```

```suggestion
  ] ++ lib.optionals (pythonOlder "3.8") [
    typing-extensions
    importlib-metadata
```

see [here](https://github.com/collerek/ormar/blob/master/pyproject.toml#L59-L61) and [here](https://github.com/collerek/ormar/blob/master/pyproject.toml#L63-L65).